.golangci.yml

```diff
@@ -0,0 +1,124 @@
+---
+run:
+  build-tags:
+    - selinux
+  concurrency: 6
+  deadline: 5m
+linters:
+  disable-all: true
+  enable:
```

Comment:
I prefer to invert the logic. Enable all by default and disable selectively. It's easier to maintain as we fetch new linters on updates and we actually know which linters are not working.

Reply:
Yep, changed it to an inverting logic.

Reply:
Awesome, thanks!